erpnext/stock/reorder_item.py

```diff
@@ -199,18 +200,16 @@ def send_email_notification(mr_list):
 
 def notify_errors(exceptions_list):
 	subject = "[Important] [ERPNext] Auto Reorder Errors"
-	content = """Dear System Manager,
-
-An error occured for certain Items while creating Material Requests based on Re-order level.
-
-Please rectify these issues:
----
-<pre>
-%s
-</pre>
----
-Regards,
-Administrator""" % ("\n\n".join(exceptions_list),)
+	content = """Dear System Manager, <br> An error occured for certain Items while creating Material Requests based on Re-order level.
```

Comment:
Email messages should also be translated?

Reply:
done